app/src/main/res/layout/activity_main.xml

```diff
@@ -15,9 +15,60 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:text="Hello World!"
-            app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintLeft_toLeftOf="parent"
             app:layout_constraintRight_toRightOf="parent"
             app:layout_constraintTop_toTopOf="parent"/>
 
+    <TextView
+        android:id="@+id/departure_text"
+        android:textSize="25sp"
+        android:layout_margin="5sp"
+        android:gravity="start"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/departure_text"
+        app:layout_constraintBottom_toTopOf="@id/departure_station"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"/>
+
+    <Spinner
+        android:id="@+id/departure_station"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.3"/>
```

Comment:
Was there any reason for constraining the top and bottom of this to its parent, and using a vertical bias, rather than constraining it to the elements around it? Its position is actually mostly being determined here by the top constraint on the arrival_text element and the bottom constraint on the departure_text element.

Reply:
This is something left over from when I just had the two dropdown spinners and just wanted them spaced apart on the page - I'll update it to reference the elements around it.